physionet-django/console/templates/console/training_list.html

```diff
@@ -40,12 +46,16 @@
       <div class="tab-content">
         {% if trainings %}
           {% if status == 'review' %}
+            <input type="search" oninput="search('{% url "training_list" status="review" %}', value);" placeholder="Search...">
```

Comment:
I'd render this input only once, under `if trainings` but above all the status conditions. Then just do:
```html
<input
  type="search"
  oninput="search('{% url "training_list" status=status %}', value);"
  placeholder="Search..."
>
```

Reply:
Good point, thanks!